app/config/services/platform.xml

```diff
@@ -299,6 +298,12 @@
             <argument type="service" id="app.repository.event_registration"/>
         </service>
 
+        <service id="AppBundle\Event\EventRegistrationSubscriber">
```

Comment:
Please add new definitions in `services.xml` where the tag is not needed thanks to `autoconfigure`.